cmd/update-main_test.go

```diff
@@ -136,6 +158,21 @@ func TestGetCurrentReleaseTime(t *testing.T) {
 	}
 }
 
+// Tests if the environment we are running is in kubernetes.
+func TestIsKubernetes(t *testing.T) {
+	os.Setenv("KUBERNETES_SERVICE_HOST", "10.11.148.5")
+	kubernetes := IsKubernetes()
+	if !kubernetes {
+		t.Fatalf("We should be running in kubernetes. Expected %t, got %t", true, kubernetes)
```

Comment:
The text `We should be running in kubernetes` doesn't convey the message clearly.  I think `Expected... Got...` is good enough.  But its your call.